Makefile

```diff
@@ -0,0 +1,100 @@
+.PHONY: clean clean-test clean-pyc clean-build docs help
```

Comment:
Please add documentation to the README on this makefile.

Reply:
At the moment it is not really functional other than for running the tests. This application is a bit different than the other SEAMM ones, and we are not yet looking for PEP-8 style, etc. which is where the Makefile becomes more useful. 

I intend to fix it slowly -- in the meantime you probably shouldn't use it. The makefile itself is self-documenting ... just 'make' or 'make help' gives the documentation. Once I make sure whatever is in it works, we can mention it in the README. There is no need to use it -- whatever you do now works fine -- it just makes life a bit easier and more consistent, and helps make e.g. releases which you do infrequently and so tend to forget the magic incantation.

Speaking of which, this could be a place for me to hide the detail of the chrome download ...